core/src/main/scala/org/apache/spark/scheduler/TaskDescription.scala

```diff
@@ -52,8 +55,26 @@ private[spark] class TaskDescription(
     val addedFiles: Map[String, Long],
     val addedJars: Map[String, Long],
     val properties: Properties,
-    val serializedTask: ByteBuffer) {
-
+    // Task object corresponding to the TaskDescription. This is only defined on the master; on
+    // the worker, the Task object is handled separately from the TaskDescription so that it can
+    // deserialized after the TaskDescription is deserialized.
```

Comment:
1) master -> driver
2) worker -> executor
3) ... so that it can be deserialized ... (missing "be")